app/allocation/controllers/view.js

```diff
@@ -8,8 +8,10 @@ function viewAllocation(req, res) {
   const bannerStatuses = ['cancelled']
   const movesWithoutProfile = moves.filter(move => !move.profile)
   const movesWithProfile = moves.filter(move => move.profile)
+  const { location_type: locationType } = req.location
   const moveToCardComponent = presenters.moveToCardComponent({
     showStatus: false,
+    locationType,
```

Comment:
After seeing the end-to-end failure you pointing out I think this should in fact be (along with other suggestion above):
```suggestion
    locationType: fromLocation.location_type,
```